Makefile

```diff
@@ -229,8 +229,8 @@ metallb:
 	cat hack/metallb.yaml | sed -E "s|172.19|$$(docker network inspect -f '{{range .IPAM.Config}}{{.Gateway}}{{end}}' kind | sed -E 's|^([0-9]+\.[0-9]+)\..*$$|\1|g')|g" | kubectl apply -f -
 
 cert-manager:
-	$(HELM) repo add bitnami https://charts.bitnami.com/bitnami
-	$(HELM) upgrade --install cert-manager bitnami/cert-manager --namespace certmanager-system --create-namespace --set "installCRDs=true"
+	$(HELM) repo add jetstack https://charts.jetstack.io
+	$(HELM) upgrade --install cert-manager jetstack/cert-manager --namespace certmanager-system --create-namespace --set "installCRDs=true"
```

Comment:
QQ: what's wrong with Bitnami's chart?

Reply:
Looks like they have a broken dependency or something. Either way, bitnami charts tend to have some extra stuff which isn't really needed in most cases (IMO).

https://github.com/clastix/kamaji/actions/runs/12776386026/job/35614933651

```
Release "cert-manager" does not exist. Installing it now.
index.go:346: skipping loading invalid entry for chart "airflow" "" from /home/runner/.cache/helm/repository/bitnami-index.yaml: validation: chart.metadata.version is required
Error: failed to download "bitnami/cert-manager"
```